### PR TITLE
fix for vanishing other avatars

### PR DESCRIPTION
**Pull request by AndrewMeadows**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/12154/HighFidelity-Beta-PR13835-90429f0.apk

**Comment by AndrewMeadows** [comment]
This PR is a fix for a bug causing other avatars to sometimes vanish from view, often mysteriously depending on the viewer's camera orientation, as documented in the internal manuscript case 17632:
https://highfidelity.manuscript.com/f/cases/17632/Avatars-keep-disappearing-from-view

**Test Plan**:
The bug is intermittent.  Sometimes it happens immediately, sometimes it takes a while to trigger it.  Here is the repro recipe I developed while hunting for where it was introduced:

**(1)** Login two users: UserA and UserB.

**(2)** Walk UserA around in a big circle, about 5 - 10m in diameter.

**(3)** Have the UserB follow UserA but behind in phase by about pi/2, such that UserB always sees UserA walking away to the left (for CCW) or right (for CW).

**(4)** Have UserB watch out for a case where UserA incorrectly (perhaps momentarily) vanishes (but their shadow will probably still be visible if shadows are enabled).
**-->** this will happen on `hifi/master`
**-->** this should NOT happen in this PR

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12154/HighFidelity-Beta-PR13835-90429f0.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12154/HighFidelity-Beta-Interface-PR13835-90429f0.dmg

File: interface/src/avatar/AvatarManager.cpp [hunk 1]
```diff
@@ -277,6 +277,9 @@ void AvatarManager::updateOtherAvatars(float deltaTime) {
         sortedAvatars.pop();
     }
 
+    if (_shouldRender) {
+        qApp->getMain3DScene()->enqueueTransaction(transaction);
+    }
```
Comment (AndrewMeadows) [inline interface/src/avatar/AvatarManager.cpp]: The **use** of the transaction was accidentally deleted in a previous PR (13785).

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12154/HighFidelity-Beta-PR13835-90429f0.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12154/HighFidelity-Beta-Interface-PR13835-90429f0.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Avatars do not flicker and disappear. This Fix has worked.

**Pull request merged**